### PR TITLE
Feature/3157/remove-download-button

### PR DESCRIPTION
**Pull request by Nereboss**
# Remove depricated download button

Closes: #3157 

## Description

Removes the download button from the UI and all corresponding files that are no longer necessary.

### CODE REVIEW

**Comment by Nereboss** [comment]
One acceptance criteria was that the changelog informs about this "breaking" change. Did this happen in a sufficient manner or does the changelog need to be adjusted?

File: gh-pages/_docs/06-02-user-controls.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The controls that are available to you are:
...
 | --- | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | 1   | Settings        | Load cc.json map or multiple maps (_hold shift_).                                                                                                                                                                                                                |
-| 2   | Settings        | Download the current map as a JSON-file. **Deprecation Warning!** This feature will soon be removed. It is already replaced by the controls nr. 15 and 16.                                                                                                       |
+| 2   | Settings        | **Removed!** This button no longer exists and has been replaced by the controls Nr. 15 and 16.                                                                                                                                                                   |
```
Comment (ce-bo) [inline gh-pages/_docs/06-02-user-controls.md:18]: Let's remove this line completely :-)
```suggestion
```

Comment (ce-bo) [reply gh-pages/_docs/06-02-user-controls.md:18]: Ok, I see that there is a gap then. So it is fine for me to keep it until we update the screenshot.

**Review by ce-bo — APPROVED** [review]
LGTM :-)

**Pull request merged**